<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegralLeftRectFrame</class>
 <widget class="QWidget" name="IntegralLeftRectFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="inputGroupBox">
       <property name="title">
        <string>Входные данные</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,4,0" columnminimumwidth="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="iterationLabel">
            <property name="text">
             <string>Итерации :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="limitLabel">
            <property name="text">
             <string>Пределы :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSpinBox" name="iterationSpinBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="functionEdit">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="bLabel">
              <property name="text">
               <string>a :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="limitAEdit">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="aLabel">
              <property name="text">
               <string>b :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="limitBEdit">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="functionLabel">
            <property name="text">
             <string>Формула :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QGroupBox" name="methodGroupBox">
            <property name="title">
             <string>Метод</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="3" column="0">
              <widget class="QRadioButton" name="rightRectRadioButton">
               <property name="text">
                <string>Метод правых прямоугольников</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="parabolicRadioButton">
               <property name="text">
                <string>Метод парабол</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="leftRectRadioButton">
               <property name="text">
                <string>Метод левых прямоугольников</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="midRectRadioButton">
               <property name="text">
                <string>Метод серединных прямоугольников</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="trapezoidRadioButton">
               <property name="text">
                <string>Метод трапеций</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="outputGroupBox">
       <property name="title">
        <string>Ответ</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="answerEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>∫f(x) =</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
